src/abstractdataframe/abstractdataframe.jl

```diff
@@ -334,16 +334,22 @@ end
 Report descriptive statistics for a data frame
 
 ```julia
-describe(df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissing, :nunique, :eltype])
+describe(df::AbstractDataFrame)
+describe(df::AbstractDataFrame, stats::Union{Symbol, Pair{Symbol}}...)
 ```
 
 **Arguments**
 
-* `df` : the AbstractDataFrame
-* `stats::Union{Symbol,AbstractVector{Symbol}}` : the summary statistics to report. If
-  a vector, allowed fields are `:mean`, `:std`, `:min`, `:q25`, `:median`,
-  `:q75`, `:max`, `:eltype`, `:nunique`, `:first`, `:last`, and `:nmissing`. If set to
-  `:all`, all summary statistics are reported.
+* `df` : the `AbstractDataFrame`
+* `stats::Union{Symbol, Pair{Symbol}}...` : the summary statistics to report. 
+  Arguments can be: 
+    *  A symbol from the list `:mean`, `:std`, `:min`, `:q25`, 
+      `:median`, `:q75`, `:max`, `:eltype`, `:nunique`, `:first`, `:last`, and 
+      `:nmissing`. The default statistics used
+      are `:mean`, `:min`, `:median`, `:max`, `:nunique`, `:nmissing`, and `:eltype`.
+    * `:all` as the only `Symbol` argument to return all statistics. 
+    * Finally, users can provide their own functions in the form of a 
```

Comment:
"a ... pairs"

I'd also remove "users can provide their own functions": it breaks the logic of the bullets and it's quite verbose. Just say that `function` is a custom function. Also for `name` it could be useful to say that it's the name of the resulting column.

Reply:
fixed.